### PR TITLE
Adjust matchers for newest luatest version

### PR DESCRIPTION
**Pull request by RunsFor**
assertWithSomeMatcher -> assert_with_some_matcher

Make tests pass with latest luatest version (0.3.0)

### CODE REVIEW

File: test/integration/admin_test.lua [hunk 1]
```diff
@@ -61,7 +61,7 @@ function g.test_api()
         })
 
     local auth_params = resp['data']['cluster']['auth_params']
-    t.assert_true(auth_params['enabled'])
+    t.assert(auth_params['enabled'])
```
Comment (rosik) [inline test/integration/admin_test.lua]: essert_equals true. return type is important

File: test/integration/admin_test.lua [hunk 2]
```diff
@@ -144,7 +144,7 @@ end
 function g.test_login()
     local login = function(username, password)
         local auth_data = string.format("username=%s&password=%s", username, password)
-        local res = g.server:http_request('post', '/login', {body = auth_data, raw = true})
+        local res = g.server:http_request('post', '/login', {body = auth_data, raise = false})
```
Comment (rosik) [inline test/integration/admin_test.lua]: Then update luatest version in CI deps (and in readme). Without it CI fails.

File: test/integration/api_edit_test.lua [hunk 3]
```diff
@@ -275,7 +275,7 @@ local function test_all_rw(all_rw)
 
     for _, srv in pairs(replicaset['servers']) do
         if srv['uuid'] == replicaset['master']['uuid'] then
-            t.assert_false(srv['boxinfo']['general']['ro'])
+            t.assert_not(srv['boxinfo']['general']['ro'])
```
Comment (rosik) [inline test/integration/api_edit_test.lua]: assert_equals

File: test/integration/api_join_test.lua [hunk 4]
```diff
@@ -62,7 +62,7 @@ function g.test_join_server()
             )
         }]]
     })
-    t.assert_true(resp['data']['probe_server'])
+    t.assert(resp['data']['probe_server'])
```
Comment (rosik) [inline test/integration/api_join_test.lua]: assert_equals

Comment (RunsFor) [reply test/integration/api_join_test.lua]: ✅

File: test/integration/api_edit_test.lua (hunk 3 above)
Comment (RunsFor) [reply test/integration/api_edit_test.lua]: ✅

**Pull request merged**